scanner/src/main/kotlin/ScannerWrapperConfig.kt

```diff
@@ -31,12 +31,12 @@ data class ScannerWrapperConfig(
     val matcherConfig: ScannerMatcherConfig,
 
     /**
-     * If `true`, scan results of this scanner shall be written to the configured scan storages.
+     * If `true`, scan results of this scanner shall be read from the configured scan storages.
```

Comment:
The docs fixes here and below should be squashed to the commit where they were introduced.